CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Added `create_snapshot_artifacts` devtool command that creates snapshot
+  artifacts for microVMs built from config file.
```

Comment:
I wouldn't add a changelog entry since this is a tool to be used only for development/testing purposes

Reply:
Makes sense. Removed.